src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,23 @@
+package calculator;
+
+public class Operator {
+
+    public static int add(int left, int right){
+        return left + right;
+    }
+
+    public static int subtract(int left, int right){
+        return left - right;
+    }
+
+    public static int multiply(int left, int right){
+        return left * right;
+    }
+
+    public static int divide(int left, int right){
+        if(right == 0)
+            throw new IllegalArgumentException("0으로 나눌 수 없습니다.");
```

Comment:
```suggestion
        if (right == 0) {
            throw new IllegalArgumentException("0으로 나눌 수 없습니다.");
        }
```
중괄호는 생략하지 않는것이 컨벤션 입니다. 
https://google.github.io/styleguide/javaguide.html#s4.1-braces